tasks/specs/linear-904f74a5-e1b6-4740-a0df-8c92ec73314b.md

```
@@ -0,0 +1,71 @@
+- `codex plugin marketplace add`
+- `codex plugin marketplace remove`
+- `codex marketplace add`
+- `README.md`
+- `docs/public/downstream-setup.md`
+- `scripts/pack-smoke.mjs`
+- `tests/pack-smoke.spec.ts`
+- `plugins/codex-orchestrator/launcher.mjs`
+- `CO-196`
+
+## Current / Reference / Target Parity Matrix
+| Surface | Current (pre-lane) | Reference (Codex 0.122) | Target (this lane) |
```

markdownlint-cli2 (0.22.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)